config/lava/chromeos/chromeos-flash-template.jinja2

```diff
@@ -0,0 +1,83 @@
+{% extends 'base/kernel-ci-base.jinja2' %}
+{% block main %}
+{{ super () }}
+{% endblock %}
+
+{% block actions %}
+{{ super () }}
+
+actions:
+- deploy:
+    kernel:
+      url: {{ flashing_kernel }}
+    modules:
+      url: {{ flashing_modules }}
+      compression: xz
+    nfsrootfs:
+      url: {{ flashing_rootfs }}
+      compression: xz
+    ramdisk:
+      url: {{ flashing_ramdisk }}
+      compression: gz
+    os: oe
+    timeout:
+      minutes: 15
+    to: tftp
+
+
+- boot:
+    timeout:
+      minutes: 15
+    method: depthcharge
+    commands: nfs
+    prompts:
+      - '/ #'
+
+- test:
+    timeout:
+      minutes: 5
+    docker:
+      image: {{ docker_image }}
+      wait:
+        device: true
+    definitions:
+    - repository:
+        metadata:
+          format: Lava-Test Test Definition 1.0
+          name: docker-network
+        run:
+          steps:
+            - lava-test-case uname --shell "uname -a"
+            - ping -c 1 172.17.0.1
+            - wget http://172.17.0.1/chromeos/{{ chromeos_image }}
```

Comment:
We could also treat the `wget` command as a test case to show in the results whether it worked or not, like the `scp` command.